content/hi/docs/tutorials/hello-minikube.md

```diff
@@ -0,0 +1,306 @@
+
+आमतौर पर, डैशबोर्ड केवल आंतरिक कुबेरनेट्स वर्चुअल नेटवर्क के भीतर से ही पहुँचा जा सकता है।
+डैशबोर्ड को कुबेरनेट्स वर्चुअल नेटवर्क के बाहर से एक्सेस करने योग्य बनाने के लिए `dashboard` कमांड एक अस्थायी प्रॉक्सी बनाता है।
+
+प्रॉक्सी को रोकने और प्रक्रिया से बाहर निकलने के लिए `Ctrl+C` का प्रयोग करें।
+कमांड से बाहर निकलने के बाद, डैशबोर्ड कुबेरनेट्स क्लस्टर में चलता रहता है। 
+आप डैशबोर्ड तक पहुंचने के लिए एक और प्रॉक्सी बनाने के लिए फिर से `dashboard` कमांड चला सकते हैं।
+{{< /note >}}
+
+## URL से डैशबोर्ड खोलें {#open-dashboard-with-url}
+
+यदि आप वेब ब्राउज़र नहीं खोलना चाहते हैं, तो URL प्राप्त करने के लिए url फ़्लैग के साथ `dashboard` कमांड चलाएँ:
+
+```shell
+minikube dashboard --url
+```
+
+## डेप्लॉयमेंट बनाएँ
+
+कुबेरनेट्स [*पॉड*](/hi/docs/concepts/workloads/pods/) एक या अधिक कंटेनरों का एक समूह है, 
+जो प्रशासन और नेटवर्किंग के उद्देश्यों के लिए एक साथ बंधे होते हैं। इस ट्यूटोरियल के 
+पॉड में केवल एक कंटेनर है। कुबेरनेट्स 
+[*डेप्लॉयमेंट*](/hi/docs/concepts/workloads/controllers/deployment/) आपके पॉड के स्वास्थ्य की 
+जाँच करता है और यदि पॉड बंद हो जाता है तो पॉड के कंटेनर को पुनः आरंभ करता है।
+पॉड्स के निर्माण और स्केलिंग को प्रबंधित करने के लिए डेप्लॉयमेंट अनुशंसित तरीका है।
+
+1. पॉड को प्रबंधित करने वाला डेप्लॉयमेंट बनाने के लिए `kubectl create` कमांड का उपयोग करें। पॉड 
+प्रदान की गई डॉकर इमेज के आधार पर एक कंटेनर चलाता है।
+
+    ```shell
+    kubectl create deployment hello-node --image=k8s.gcr.io/echoserver:1.4
+    ```
+
+2. डेप्लॉयमेंट देखें:
+
+    ```shell
+    kubectl get deployments
+    ```
+
+    आउटपुट कुछ इस समान होगा:
+
+    ```
+    NAME         READY   UP-TO-DATE   AVAILABLE   AGE
+    hello-node   1/1     1            1           1m
+    ```
+
+3. पॉड देखें:
+
+    ```shell
+    kubectl get pods
+    ```
+
+    आउटपुट कुछ इस समान होगा:
+
+    ```
+    NAME                          READY     STATUS    RESTARTS   AGE
+    hello-node-5f76cf6ccf-br9b5   1/1       Running   0          1m
+    ```
+
+4. क्लस्टर इवेंट देखें:
+
+    ```shell
+    kubectl get events
+    ```
+
+5. `kubectl` कॉन्फ़िगरेशन देखें:
+
+    ```shell
+    kubectl config view
+    ```
+
+{{< note >}}
+`kubectl` कमांड के बारे में अधिक जानकारी के लिए [kubectl अवलोकन](/docs/reference/kubectl/overview/) देखें।
+{{< /note >}}
+
+## सर्विस बनाएं
+
+आमतौर पर, पॉड कुबेरनेट्स क्लस्टर के भीतर अपने आंतरिक IP पते से ही पहुँचा जा सकता है।
+`hello-node` कंटेनर को कुबेरनेट्स वर्चुअल नेटवर्क के
+बाहर से सुलभ बनाने के लिए,पॉड को
+कुबेरनेट्स [*Service*](/docs/concepts/services-networking/service/)(सर्विस) के रूप में बेनकाब करना होगा।
+
+1. `kubectl expose` कमांड का उपयोग करके पॉड को सार्वजनिक इंटरनेट पर एक्सपोज़ करें:
+
+    ```shell
+    kubectl expose deployment hello-node --type=LoadBalancer --port=8080
+    ```
+
+    `--type=LoadBalancer` फ्लैग इंगित करता है कि आप क्लस्टर के बाहर 
+    अपने सर्विस को प्रदर्शित करना चाहते हैं।
+    
+    इमेज के अंदर एप्लिकेशन कोड `k8s.gcr.io/echoserver` केवल TCP पोर्ट 8080 पर सुनता है। 
```

Comment:
can we have an alternative for पर सुनता है?

Reply:
Can you suggest something, we need to convey "listens on TCP port 8080"

Reply:
I think we can use either ` केवल TCP पोर्ट 8080 को सुनता हैं।`  **or**  `केवल TCP पोर्ट 8080 पे सुनता हैं।` for this.  I also feel it will be  okay even if we go with the original sentence. Thoughts?